data/heb0001/heb010/__cts__.xml

```diff
@@ -6,6 +6,7 @@
     </cpt:structured-metadata>
   <ti:translation xml:lang= "eng" urn="urn:cts:hebrewlit:heb0001.heb010.1st1K-eng1" workUrn="urn:cts:hebrewlit:heb0001.heb010">
     <ti:label xml:lang="lat">Isaias</ti:label>
-    <ti:description xml:lang="mul">Hebrew Bible, Isaias, Ottley, Cambridge, 1904</ti:description>
+    <ti:description xml:lang="mul">Septuaginta. The Book of Isaiah According to the Septuagint (Codex Alexandrinus). Ottley, Richard, Rusden, editor. 
```

Comment:
I would not include Septuginta here as this is apart from that.

Reply:
Will change this.

Reply:
And you are right @lcerrato there was additional metadata in the cts_.xml file. ` <cpt:structured-metadata>
    <dct:hasVersion>urn:cts:greekLit:tlg0527.tlg048</dct:hasVersion>
    </cpt:structured-metadata>` I guess we will just leave this as is for now, it's just without the matching URNs there is no way to align the texts and I could see someone wanting to align the two English translations.

Reply:
@AlisonBabeu I'm not opposed to renaming, I'm just wondering how it would work since we can't currently align with either of these: https://scaife.perseus.org/library/urn:cts:ancJewLit:hebBible.isaiah/